packages/atlas/src/components/Banner/Banner.tsx

```diff
@@ -8,7 +8,7 @@ import { BannerDescription, BannerHeader, BannerText, BannerWrapper, CloseButton
 export type BannerProps = {
   id: string
   title?: string
-  description?: string
+  description?: string | React.ReactNode
```

Comment:
`string` is unnecessary here, `ReactNode` contains `ReactText` which is `string | number`